main.go

```diff
@@ -75,9 +66,9 @@ func main() {
 
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
 		Scheme:             runtimescheme,
-		MetricsBindAddress: metricsAddr,
+		MetricsBindAddress: config.MetricsAddress,
 		Port:               9443,
-		LeaderElection:     enableLeaderElection,
+		LeaderElection:     false,
```

Comment:
hmm, we never tested it, but what if somebody would like to try to execute multiple k8gb controllers with leader election mechanism in the future? currently it is optional but here it is becoming hardcoded.

Reply:
enableLeaderElection was accessible through an argument of executable. Making such functionality available would mean passing the argument to the controller pod. Till now it was always false anyway.

What we can do is create an issue, expose an environment variable and test.

Reply:
@kuritka ok let's log the issue then

Reply:
even more, I think it requires to have locking to be implemented.

Reply:
good point @k0da. Feel free and add your comments by #526 please.